modules/core/src/test/java/org/apache/ignite/internal/processors/cache/datastructures/IgniteCollectionAbstractTest.java

```diff
@@ -148,6 +148,6 @@ protected static GridCacheContext cctx(IgniteSet set) {
         if (set instanceof GridCacheSetProxy)
             return GridTestUtils.getFieldValue(set, GridCacheSetProxy.class, "cctx");
         else
-            return GridTestUtils.getFieldValue(set, GridCacheSetImpl.class, "ctx");
+            return GridTestUtils.getFieldValue(set, GridCacheSet.class, "ctx");
```

Comment:
Will it work if we revert changes?

Reply:
If we will place current GridCacheSetImpl into inner class of current GridCacheSet.